Modix.Services/Giveaways/WinnersResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Immutable;
+
+namespace Modix.Services.Giveaways
+{
+    public class WinnersResult
+    {
+        public WinnersResult(ImmutableArray<ulong> winners)
+        {
+            IsError = false;
+            Winners = winners;
+        }
+
+        public WinnersResult(string error)
+        {
+            IsError = true;
+            Error = error;
+        }
+
+        public ImmutableArray<ulong> Winners { get; }
```

Comment:
Does `ImmutableArray<T>` give us any particular advantage over `.ToArray()` cast as `IReadOnlyCollection<T>` or `IReadOnlyList<T>`?

Reply:
Not strictly. Just the potential piece of mind that the collection can't accidentally be modified. I can go either way with this though.

Reply:
Same.